checkov/common/runners/runner_registry.py

```diff
@@ -105,6 +105,9 @@ def filter_runner_framework(self):
         self.runners = filtered_runners
         return
 
+    def remove_runner(self, runner):
+        del self.runners[self.runners.index(runner)]
```

Comment:
using `remove()` here is no need for checking the index, but this method can result in a `ValueError` if the runner is not in the list.
```suggestion
    def remove_runner(self, runner):
        if runner in self.runners:
            self.runners.remove(runner)
```
or catch the `ValueError` with try ... except ... 😄